public/app/features/expressions/components/Threshold.tsx

```diff
@@ -476,7 +476,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
     font-size: ${theme.typography.bodySmall.fontSize};
     text-transform: uppercase;
     padding: 0 ${theme.spacing(1)};
-    border: 1px solid ${theme.colors.border.medium};
```

Comment:
after this change :
<img width="643" alt="Screenshot 2023-10-26 at 10 17 27" src="https://github.com/grafana/grafana/assets/33540275/d701db93-3ac3-45f8-a399-18a6a79bb5b2">